system-test/storage.ts

```diff
@@ -880,18 +880,19 @@ describe('storage', () => {
 
     let bucket: Bucket;
 
-    before(() => {
-      bucket = storage.bucket('storage-library-test-bucket');
+    before(async () => {
+      [bucket] = await storage.createBucket(generateName());
     });
 
     // Normalization form C: a single character for e-acute;
     // URL should end with Cafe%CC%81
-    it('should not perform normalization form C', () => {
+    it('should not perform normalization form C', async () => {
       const name = 'Caf\u00e9';
-      const file = bucket.file(name);
-
       const expectedContents = 'Normalization Form C';
 
+      const file = bucket.file(name);
+      await file.save(expectedContents);
+
       return file
```

Comment:
Should this logic be switched to use `await` as well?

Reply:
Yes :) It's a wonder this test wasn't flaky before!